director/tests/test_urls.py

```diff
@@ -0,0 +1,266 @@
+        anon=signin,
+        joe=[title("Account  : Teams"), link(reverse("account_create"))],
+    ),
+    check(
+        "/joe-personal-account",
+        anon=signin,
+        joe=title("Account joe-personal-account")
+    ),
+    check(
+        "/joe-personal-account/members",
+        anon=signin,
+        joe=title("Account joe-personal-account : Members"),
+    ),
+    check(
+        "/joe-personal-account/teams",
+        anon=signin,
+        joe=title("Account 1 : Teams")
+    ),
+    check(
+        "/joe-personal-account/settings",
+        anon=signin,
+        joe=title("Account 1 : Settings")
+    ),
+    check(
+        "/joe-personal-account/subscriptions",
+        anon=signin,
+        joe=title("Account joe-personal-account : Subscriptions"),
+    ),
+    check(
+        "/joe-personal-account/subscriptions/add",
+        anon=signin,
+        joe="plan"
+    ),
+    check(
+        "/projects",
+        anon=title("Projects"),
+        joe=[title("Projects"), link(reverse("project_create"))],
+    ),
+    # Most (all?) of the following tests generate the warning for all(?) users:
+    #  warnings.warn("ProjectPermissionsMixin GET", DeprecationWarning)
+    check(
+        "/joe-personal-account/public-project",
+        anon=title("public-project : Overview"),
+        joe=title("public-project : Overview"),
+        mary=title("public-project : Overview")
+    ),
+    check(
+        "/joe-personal-account/private-project",
+        anon=403,
+        joe=title("private-project : Overview"),
+        mary=403
+    ),
+    check(
+        "/joe-personal-account/private-project/files",
+        anon=403,
+        joe=title("Project private-project: Files"),
+        mary=403
+    ),
+    check(
+        "/joe-personal-account/private-project/snapshots",
+        anon=403,
+        joe=title("Project private-project: Snapshots"),
+        mary=403
+    ),
+    check(
+        "/joe-personal-account/private-project/archives",
+        anon=403,
+        joe=title("Project 2: Files"),
+        mary=403
+    ),
+    check(
+        "/joe-personal-account/private-project/activity",
+        anon=403,
+        joe=title("Project 2: Activity"),
+        mary=403
+    ),
+    check(
+        "/joe-personal-account/private-project/sharing",
+        anon=403,
+        joe=title("Project 2 : Sharing"),
+        mary=403
+    ),
+    check(
+        "/joe-personal-account/private-project/settings/metadata",
+        anon=403,
+        joe=title("Project 2 : Settings : Metadata"),
+        mary=403
+    ),
+    check(
+        "/joe-personal-account/private-project/settings/access",
+        anon=403,
+        joe=title("Project 2 : Settings : Access"),
+        mary=403
+    ),
+]
+# fmt: on
+
+
+# The following turns warnings into errors to help debug where those
+# are being generated.
+# For finer grained control see https://docs.pytest.org/en/latest/warnings.html
+# pytestmark = pytest.mark.filterwarnings("error")
```

Comment:
@beneboy I haven't been able to isolate too well where the `DeprecationWarning` is being generated, see comment above (line 131). But uncommenting line 194 should help with that. In anycase, not a high priority.